content/docs/1.18/operator.md

```diff
@@ -971,7 +971,9 @@ spec:
           - key: log_level
             path: log_level
 ```
-
+{{< warning >}}
```

Comment:
I'm wondering if the troubleshooting guide would be better suitable for this... What do you think? 

https://www.jaegertracing.io/docs/1.18/troubleshooting/